common/v2/features/Dashboard/components/AccountList.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
-import { Address, CollapsibleTable, Icon, Network, Typography, Button } from '@mycrypto/ui';
-
-import DashboardPanel from './DashboardPanel';
-import './AccountList.scss';
-import { ExtendedAccount } from 'v2/services';
+import { Address, Button, CollapsibleTable, Icon, Network, Typography } from '@mycrypto/ui';
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import styled from 'styled-components';
 import { truncate } from 'v2/libs';
+import { ExtendedAccount } from 'v2/services';
+import './AccountList.scss';
+import DashboardPanel from './DashboardPanel';
```

Comment:
Can we keep these two import groups separate? It looks like your editor auto formatted them.